PurchasesTests/Caching/InMemoryCachedObjectTests.swift

```diff
@@ -0,0 +1,124 @@
+//
+// Created by RevenueCat on 2/4/20.
+// Copyright (c) 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+import Nimble
+import XCTest
+
+@testable import Purchases
+
+class InMemoryCachedObjectTests: XCTestCase {
+
+    // mark: isCacheStale
```

Comment:
I would add a test that tests if the cache is stale when there's nothing cached

Reply:
sounds good, added